cogs/deadline.py

```diff
@@ -194,30 +191,23 @@ async def deleteReminder_error(self, ctx, error):
     #          returns a success message indicating that the reminder has been updated
     # -----------------------------------------------------------------------------------------------------------------
     @commands.command(name="changeduedate", pass_context=True,
```

Comment:
Should also be instructor only

```suggestion
    @commands.has_role('Instructor')
    @commands.command(name="changeduedate", pass_context=True,
```